fluid_onnx/ops.py

```diff
@@ -200,20 +213,39 @@ def depthtospace_op():
 def dropout_op(operator, block):
     inputs, attrs, outputs = op_io_info(operator)
     scale_input = [outputs['Out'][0] + '@dropout']
-    dropout_op = make_node(
+    dropout_node = make_node(
         'Dropout',
         inputs=inputs['X'],
         outputs=scale_input + outputs['Mask'],
         is_test=attrs['is_test'],
         ratio=attrs['dropout_prob'])
 
     # Fluid and ONNX use different dropout formula
-    scale_op = make_node(
-        'Scale',
-        inputs=scale_input,
-        outputs=outputs['Out'],
-        scale=1.0 - attrs['dropout_prob'])
-    return (dropout_op, scale_op)
+    if __onnx_ver__ == '1.0.1':
```

Comment:
How about we temporarily fix the version number to 1.0.1 right now? because there are several releases, and I have not checked the changes in the definition of operators. 

You got the point. Yes, we need to scale the output of Dropout op, otherwise inference wouldn't get right result.

Reply:
@kuke Okay, no worries, but we would probably need to come back to this, as it will bite as we try to make our ONNX models compatible for various backends.